package.json

```diff
@@ -40,6 +40,12 @@
         "command": "salesforcedx-vscode-offline-app.onboardingWizard",
         "title": "%extension.commands.config-wizard.title%",
         "category": "%extension.commands.config-wizard.category%"
+      },
+      {
+        "command": "salesforcedx-vscode-offline-app.configureLintingTools",
+        "title": "%extension.commands.config-linting-tools.title%",
+        "category": "%extension.commands.config-wizard.category%",
+        "when": "sfdx_project_opened"
```

Comment:
@maliroteh-sf I can't get `when` clause to not show the command if the folder opened in VSCode is not SFDX project. Do you know what I may be doing incorrectly?

Reply:
That's b/c you must define the `menus` section too. So the `contributes` section should look like this instead:
```
"contributes": {
  "menus": {
    "commandPalette": [
      {
        "command": "salesforcedx-vscode-offline-app.configureLintingTools",
        "when": "sfdx_project_opened"
      }
    ]
  },
  "commands": [
    {
      "command": "salesforcedx-vscode-offline-app.onboardingWizard",
      "title": "%extension.commands.config-wizard.title%",
      "category": "%extension.commands.config-wizard.category%"
    },
    {
      "command": "salesforcedx-vscode-offline-app.configureLintingTools",
      "title": "%extension.commands.config-linting-tools.title%",
      "category": "%extension.commands.config-wizard.category%"
    }
  ]
},
```